### PR TITLE
:sparkles: Adding ability to pass multiple rules files or dirs

### LINKED ISSUE #127: When a directory has an invalid rules file, rule parsing fails

### LINKED ISSUE #128: Add support for multiple --rules parameters.
Add support for multiple --rules parameters to support use cases where the user has more than one directory of ruleset files.  This will eliminate the burden of them having to consolidate into a single directory (tree).

### PR DESCRIPTION
**Pull request by shawn-hurley**
* Adding PFlag for CLI flags management.

fixes #127 
fixes #128

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -127,9 +134,12 @@ func validateFlags() error {
 		return fmt.Errorf("unable to find provider settings file")
 	}
 
-	_, err = os.Stat(*rulesFile)
-	if err != nil {
-		return fmt.Errorf("unable to find rule path or file")
+	for _, f := range *rulesFile {
+		fmt.Printf("f: %v", f)
```
Comment (shawn-hurley) [inline main.go:138]: Clean up

**Comment by shawn-hurley** [comment]
Partial fix for #78

**Comment by pranavgaikwad** [comment]
Fixes #128

**Comment by shawn-hurley** [comment]
fixes #130

File: main.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ const (
 
 var (
 	settingsFile      = flag.String("provider-settings", "provider_settings.json", "path to the provider settings")
-	rulesFile         = flag.String("rules", "rule-example.yaml", "filename or directory containing rule files")
+	rulesFile         = flag.StringArray("rules", []string{"rule-example.yaml"}, "filename or directory containing rule files")
```
Comment (djzager) [inline main.go:26]: Feels like a nit but `ruleFiles` might be more obviously a slice

**Review by djzager — APPROVED** [review]
One comment about the rulesfile variable name. Otherwise LGTM

**Pull request merged**